ci/test.yml

```diff
@@ -67,6 +67,12 @@ e2e web metadata:
 
 e2e web wallet:
   extends: .e2e web
+  script:
+    - yarn install --pure-lockfile --cache-folder .yarn --prefer-offline
+    - docker-compose pull
+    - docker-compose up -d trezor-user-env-unix bitcoin-regtest
+    - sleep 5
```

Comment:
sleep 5 should not be needed anymore due to this PR https://github.com/trezor/trezor-suite/pull/4584